internal/controlplane/server.go

```diff
@@ -316,6 +317,12 @@ func (s *Server) StartHTTPServer(ctx context.Context) error {
 
 	mw := otelhttp.NewMiddleware("webhook")
 
+	// Use Gorilla Mux router to handler path variables
+	gorillaMux := gorillamux.NewRouter()
```

Comment:
aren't we able to also handle variables using the `mux` from the GRPC gateway runtime we already have? Why do we need to introduce a new dependency?

https://grpc-ecosystem.github.io/grpc-gateway/docs/operations/inject_router/

Reply:
Thanks for pointing that out! Updated